styles/home.css

```diff
@@ -0,0 +1,26 @@
+.card-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 6.25rem;
+}
+
+.image-outerContainer {
+  margin: 0rem 6.25rem;
+}
+
+.image-container {
+  max-width: none;
+  width: 96%;
+}
+
+@media screen and (max-width: 1290px) {
+  .card-container {
+    flex-wrap: wrap;
+    margin: 0 3.125rem;
+  }
+}
+
+a {
```

Comment:
It's not a big issue but styling the elements directly is not really a good practice so you can consider making class for this styles.

Reply:
Thank you for pointing this out